<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMarginWidget</class>
 <widget class="QWidget" name="NewMarginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="presetLayoutLabel">
       <property name="text">
        <string>Preset Layouts:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="PresetLayout" name="presetLayoutComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="leftMarginLabel">
       <property name="text">
        <string>Left:</string>
       </property>
       <property name="buddy">
        <cstring>leftMarginSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ScrSpinBox" name="leftMarginSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="rightMarginLabel">
       <property name="text">
        <string>Right:</string>
       </property>
       <property name="buddy">
        <cstring>rightMarginSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="ScrSpinBox" name="rightMarginSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="topMarginLabel">
       <property name="text">
        <string>Top:</string>
       </property>
       <property name="buddy">
        <cstring>topMarginSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="ScrSpinBox" name="topMarginSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="bottomMarginLabel">
       <property name="text">
        <string>Bottom:</string>
       </property>
       <property name="buddy">
        <cstring>bottomMarginSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="ScrSpinBox" name="bottomMarginSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="marginLinkCheckBox">
       <property name="text">
        <string>Link</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="printerMarginsPushButton">
       <property name="text">
        <string>Printer Margins...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">ui/scrspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>PresetLayout</class>
   <extends>QComboBox</extends>
   <header location="global">ui/marginpresetlayout.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
